docs/hero-image.md

```diff
@@ -0,0 +1,38 @@
+# Guidelines For Hero Image
+
+## Strict Guidelines
+
+These are the requirements that must be followed at all costs.
+
+- Should not be NSFW.
+- Must not use material under copyright (unless permitted, like in case of a
+  collaboration).
+- Should not be detrimental to the general perception of V
+  or collaborators.
+
+## Aesthetic Guidelines
+
+These should be followed to ensure that the website's look stays consistent.
+
+- The graphic elements used must follow the [style guide](style-guide.md).
+  These include components like text, which must use stipulated fonts.
+- It is recommended to have the hero image be in landscape mode.
+- Additionally, the preferred resolution is **1600x900**.
+- It is not necessary to stick to the enforce the colour palette restrictions
+  on all images. However, it is expected that at least the elements will
```

Comment:
I do not understand the first sentence 🤔 .

Reply:
Perhaps `It is not necessary that all images use colors from the same palette.`?

Reply:
Faulty grammar. Will fix.

What I mean to say is that there is no need to keep using blue images or black images because they are the official palette colours. It is sufficient to use them for graphic elements like text, icons, just enough to maintain the consistency of the design and play nice with the underlying image.